scripts/mass_issue_worker.py

```
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+import json
+import os
+import subprocess
+import sys
+
+REPO = "organvm/peer-audited--behavioral-blockchain"
+ROOT_DIR = "/Users/4jp/Workspace/peer-audited--behavioral-blockchain"
+
+def run_cmd(cmd, cwd=ROOT_DIR):
+    res = subprocess.run(cmd, shell=True, capture_output=True, text=True, cwd=cwd)
```

Ruff (0.15.21)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)